_assets/stylesheets/elements/reset.scss

```diff
@@ -0,0 +1,6 @@
+* {
+	-webkit-margin-start: 0;
+	-webkit-margin-end: 0;
+	-webkit-margin-before: 0;
+	-webkit-margin-after: 0;
+}
```

Comment:
@jcbpl I forget which element it was applying to, but webkit was indenting some block-level element(s) through its default styling. It was just the browser that was applying the property, nothing from USWDS or ACC styling.